recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - cython
     - blas 1.1 {{ variant }}
     - openblas 0.2.18|0.2.18.*
```

Comment:
@jakirkham I want a version of caffe with openblas 0.2.18 and boost 1.62 since that's how most of packages are in conda-forge. As soon as this is merged, I'll bump openblas.